modules/vaos/app/services/vaos/appointment_service.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module VAOS
+  class AppointmentService < Common::Client::Base
+    configuration VAOS::Configuration
+
+    def get_appointments(user)
```

Comment:
I'd prefer to extract these date helpers to pvt methods, to keep the service-based `<action>_*` methods as consistent, clean and patterned as possible. Something like:

```rb
# ...
    def get_appointments(user)
      with_monitoring do
        response = perform(:get, patient_appointment_url(user), {})
        VAOS::AppointmentService::Response.parsed_response(response)
      end
    end

    private
     
    def patient_appointment_url(user)
      "/appointments/v1/patients/#{user.icn}/appointments"\
        "?startDate=#{start_date}&endDate=#{end_date}&useCache=false&pageSize=0"
    end

    def start_date
     (Time.now.utc.beginning_of_day + 7.hours).strftime('%Y-%m-%dT%TZ')
    end

    def end_date
      (Time.now.utc.beginning_of_day + 8.hours + 4.months).strftime('%Y-%m-%dT%TZ')
    end
```


Further refactoring around the parameters might be needed based on product requirements but this helps to organize the code a bit more.

Reply:
Will also make writing a spec easier/not worry about testing logic of time but just behavior